tests/policy_snapshot/snapshots/20140601

```diff
@@ -25,3 +25,4 @@ signature schemes:
 curves:
 - secp256r1
 - secp384r1
+strongly preferred groups: (empty)
```

Comment:
Can you just not output this field in our snapshots if this field is null? This is already a huge PR so that would cut down on a lot of file changes.

Reply:
I can, but I generally prefer to be explicit when possible instead of implicit as it is clearer and avoids edge cases when fields are missing.